lib/montrose/clock.rb

```diff
@@ -10,32 +10,32 @@ def initialize(opts = {})
       @every = @options.fetch(:every) { fail ConfigurationError, "Required option :every not provided" }
       @interval = @options.fetch(:interval)
       @start_time = @options.fetch(:start_time)
-      @at = @options.fetch(:at, nil)
+      @at = @options.fetch(:at, []).sort
```

Comment:
Sorting `:at` option allows us to trust the data is ordered by time so we can increment index without concern for times not being in order.